frontend/src/components/TodoList.tsx

```diff
@@ -6,14 +6,20 @@ import TodoListItem from "./TodoListItem";
 interface TodoListProps {
   todos: Todo[];
   deleteTodo: (id: string) => void;
+  checkTodo: (id: string, isDone: boolean) => void;
```

Comment:
When defining methods passed as `props`, its a convention to prefix their names with `on`, to make it clear to the user that the value passed should be a function.

Rename these props to `onDeleteTodo` and `onCheckTodo`,